### ISSUE TITLE
[Feature Request]: indicate in the playlist list which playlists the video is already in

### ISSUE THREAD

**Issue by aleksejrs** [FreeTubeApp/FreeTube#4523]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

1. Press the "+" button → there are playlist thumbnails and titles. There is no way to know if the video is in any of the playlists, except for going to the Playlists page separately and going into the playlist and looking through it (or copying the text into a text editor and searching).
2. Click the thumbnail → it gets a checkbox.
3. Click Save → the video is added to the selected playlists. Even if it already was in them.

### Proposed Solution

After step 2, show a checkbox on every playlist the video is in.
Removing the checkbox could remove the video from the playlist. In that case, playlist entries should be unique.

### Alternatives Considered

Bad alternative: hide the playlists where the video is.
Additional feature: list the playlists on the video's page.

### Issue Labels

display more information to user, ease of use improvement, improvement to existing feature

### Additional Information

v0.19.1-nightly-3781 Beta

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Closing this because playlist related issues will get transferred here soon

Duplicate of 
https://github.com/PikachuEXE/FreeTube/issues/55
